boards/arm/p8/p8.dts

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2018 Zelin <zelin@makerdiary.com>
+ * Copyright (c) 2018 makerdiary.com.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+
+PO.23 LCD backlight
+#include <nordic/nrf52832_qfaa.dtsi>
+
+ */
+
+/dts-v1/;
+#include <nordic/nrf52832_qfaa.dtsi>
+
+/ {
+	model = "p8 open source watch";
+	compatible = "nrf52832";
+
+	chosen {
```

Comment:
Could you try to format all your file ? 
There is here some issues because you are using tabs and space indentation.
Default Zephyr indentation character is tab. =)

Reply:
It should be all tabs already, I took the pinetime.dts and just changed the button pin

Reply:
It's quite possible that certain existing files are incorrectly formatted (as inherited from the project we forked). I suppose we should fix all indentation/formatting in a dedicated PR.

If it's really just the button pin that's different with P8, then maybe a devicetree overlay can be used so as to avoid code duplication.